src/style.css

```diff
@@ -0,0 +1,18 @@
+body {
+  margin: 0;
+}
+
+.box_1 {
```

Comment:
this is not a good idea to use numbers to identify the element, use names that describe the content or its behavior